test/cases/api/common/common.sh

```diff
@@ -19,6 +19,20 @@ function _instanceCheck() {
   echo "✔️ Instance checking"
   local _ssh="$1"
 
+  # Retry loop to wait for instance to be ready
+  # This is here especially because of gcp test
+  RETRIES=10
+  for i in $(seq 1 $RETRIES); do
+    echo "Attempt $i of $RETRIES: Checking instance status..."
+    if eval "$_ssh rpm -q postgresql dummy"; then
+      echo "Instance is up and ready!"
+      break
+    else
+      echo "Instance is still booting or SSH key not propagated, retrying in 30 seconds..."
+      sleep 30
+    fi
+  done
+
   # Check if postgres is installed
   $_ssh rpm -q postgresql dummy
```

Comment:
I suppose you can now drop this line

Reply:
Yeah, I was thinking that it might be more explicit this way, but it does check it in the loop and now it's ran twice in the test runs, I'll remove it.

Reply:
This is not relevant any more.